### PR TITLE
refactor: split MetadataStore contracts into OfferStore and UserStore

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/features/metadata/saga.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* global web3 */
-import MetadataStore from '../../../embarkArtifacts/contracts/MetadataStore';
-import ArbitrationLicense from '../../../embarkArtifacts/contracts/ArbitrationLicense';
+import OfferStore from '../../../embarkArtifacts/contracts/OfferStore';
+import UserStore from '../../../embarkArtifacts/contracts/UserStore';import ArbitrationLicense from '../../../embarkArtifacts/contracts/ArbitrationLicense';
```
Comment (jrainville) [inline src/js/features/metadata/saga.js]: Little oopsie here

File: test/escrow_spec.js [hunk 2]
```diff
@@ -92,20 +99,20 @@ contract("Escrow", function() {
   const feeAmount = Math.round(tradeAmount * (feePercent / 100));
 
   // util
-  let expirationTime = parseInt((new Date()).getTime() / 1000, 10) + (5 * 86400);
-  const expireTransaction = async() => {
+    const expireTransaction = async() => {
```
Comment (jrainville) [inline test/escrow_spec.js]: I think the indentation is broken here

File: src/js/features/metadata/saga.js (hunk 1 above)
Comment (richard-ramos) [reply src/js/features/metadata/saga.js]: oops!

File: test/escrow_spec.js (hunk 2 above)
Comment (richard-ramos) [reply test/escrow_spec.js]: oops x 2!

**Pull request merged**